<?xml version="1.0" encoding="utf-8"?>
<launch>
      <arg name="model" default="$(find third_arm)/urdf/third_arm_human.urdf"/>
      <arg name="gui" default="true" />
      <arg name="rvizconfig" default="$(find third_arm)/urdf/third_arm_human_rviz.rviz"/>
      
      <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
      <param name="use_gui" value="True"/>
      <rosparam param="source_list"> ["/third_arm_joints"] </rosparam >
      
      <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
      <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
      <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="false"/>
</launch>